src/domain/accounts/index.ts

```diff
@@ -65,3 +68,11 @@ export const checkedToContactAlias = (alias: string): ContactAlias | ValidationE
   }
   return alias as ContactAlias
 }
+
+export const checkedToWithdrawFee = (fee: number): WithdrawFee | ValidationError => {
+  const { min, max } = getWithdrawFeeRange()
```

Comment:
`domain` layer should be pure functions - inject the config.

Reply:
I checked to see if we were importing config in domain prior to this change, and we are in `domain/rate-limit` and `domain/wallets`. Is that planned for removal?